tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/TestingFrameworkImpactedTests.cs

```diff
@@ -243,6 +242,26 @@ private void InitGit()
                 output = RunGitCommand("branch --show-current");
             }
 
+            if (output.ExitCode == 0)
+            {
+                // Retrieve branch name
+                branch = output.Output.Trim();
+            }
+
+            if (output.ExitCode == 0)
```

Comment:
I think we can combine these checks?
```suggestion
            if (output.ExitCode == 0)
            {
                // Retrieve branch name
                branch = output.Output.Trim();

```

tbh, I think this whole method would be easier to follow with early-returns. It's a bit confusing given we have various nested `if` (many of which are unnecessary).